Castanet/Feed.php

```diff
@@ -166,6 +206,39 @@ public function setItunesBlock($itunes_block)
 	}
 
 	// }}}
+	// {{{ public function setAtomLink()
+
+	public function setAtomLink($atom_link)
+	{
+		$this->atom_link = strval($atom_link);
+	}
+
+	// }}}
+	// {{{ public function setItunesCategory()
+
+	public function setItunesCategory($itunes_category, $itunes_subcategory)
```

Comment:
subcategory is optional and multiple subcategories are supported by the spec. This may make sense as:

```php
array $itunes_subcategories = array()
```